src/helpers/helpers.js

```diff
@@ -0,0 +1,3 @@
+export const generateId = () => Math.floor(Math.random() * 1000000)
+
+export const addArrayItem = (item, array) => [...array, item]
```

Comment:
This is literally the same as `array.concat(item)`. I don't think that's complex enough to be a helper method :)